conf_sp_idp/idp/idp_conf.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+"""
+
+    conf_sp_idp.idp.idp_conf.py
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    SAML2 IDP configuration with bindings, endpoints, and authentication contexts.
+
+    This file is part of MSS.
+
+    :copyright: Copyright 2023 Nilupul Manodya
+    :copyright: Copyright 2016-2023 by the MSS team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+# Parts of the code
+
+import os.path
+
+from saml2 import BINDING_HTTP_ARTIFACT
+from saml2 import BINDING_HTTP_POST
+from saml2 import BINDING_HTTP_REDIRECT
+from saml2 import BINDING_SOAP
+from saml2 import BINDING_URI
+from saml2.saml import NAME_FORMAT_URI
+from saml2.saml import NAMEID_FORMAT_PERSISTENT
+from saml2.saml import NAMEID_FORMAT_TRANSIENT
+
+
+try:
+    from saml2.sigver import get_xmlsec_binary
+except ImportError:
+    get_xmlsec_binary = None
+
+if get_xmlsec_binary:
+    XMLSEC_PATH = get_xmlsec_binary(["/opt/local/bin"])
+else:
+    XMLSEC_PATH = '/usr/bin/xmlsec1'
```

Comment:
this likly get never reached once libxmlsec1 is a dependency on something else or we add it to the meta.yaml